requestor/worldstate.go

```diff
@@ -119,6 +121,7 @@ func DailyDeals(platform string) {
 		if eror == nil {
 			var jsonobjc gjson.Result = gjson.Parse(string(body))
 			fmt.Println(colorformat.BackGrnBold(colorformat.ForeBlkBold("PRUDENCE/DEALS")))
+			printNumberOfRecordsFetched(len(jsonobjc.Array()))
```

Comment:
Please print the number of records fetched after all the results are displayed.

Reply:
Acknowledged, would change the same.